files/poketool/icongra/poke_icon/poke_icon.mk

```diff
@@ -0,0 +1,36 @@
+POKE_ICON_DIR := files/poketool/icongra/poke_icon
+POKE_ICON_NARC := $(POKE_ICON_DIR)/poke_icon.narc
+POKE_ICON_BUILD_DIR := $(POKE_ICON_DIR)/build-poke_icon
+
+POKE_ICON_GFX_FLAGS_ICON := -clobbersize -version101
```

Comment:
these are already defined in graphics_rules.mk

Reply:
Trying to use them legitimately breaks the entire build. I'm not sure exactly what I did wrong to make them not be visible, but they are functionally unusable.

Reply:
see pika's PR for an example on how to do this ( #296 )

Reply:
I tried adding these lines, but that just broke my targets, and the NARC would no longer build:

```
CLOBBER_SIZE_VERSION101_NCGR_FILES += $(POKE_ICON_ICON_OBJS)
4BPP_NCLR_FILES += $(POKE_ICON_PAL_OBJS)
```

As best as I can tell, these targets are unused save for a single instance in the Unown puzzle app build, so I would really rather this not be what holds up this PR.